tasks/test_jest.js

```diff
@@ -3,15 +3,18 @@ const { resolve } = require('path');
 module.exports = function (grunt) {
   grunt.registerTask('test:jest', function () {
     const done = this.async();
-    runJest().then(done, done);
+    runJest(resolve(__dirname, '../scripts/jest.js')).then(done, done);
   });
 
-  function runJest() {
+  grunt.registerTask('test:jest_integration', function () {
```

Comment:
__note:__ I tried to avoid creating of `test_jest_integration.js` and duplicating the error handling logic, but tell me if you prefer a dedicated file  :)